File: server/ggGURPS.Models/Characters/Character.cs
using System.Collections.Generic;

public class Character
{
    public int Id { get; set; }
    public string Name { get; set; }

    // Attributes
    public int Strength { get; set; }
    public int Dexterity { get; set; }
    public int Inteligence { get; set; }
    public int Health { get; set; }
    public int Perception { get; set; }
    public int Will { get; set; }
    public int Magery { get; set; }
    public int DamageReduction { get; set; }
    public int HitPoints { get; set; }
    public int FatiguePoints { get; set; }
    public int EnergyReserves { get; set; }


    public ICollection<Item> Items { get; set; }
    public ICollection<Skill> Skills { get; set; }
    public ICollection<Spell> Spells { get; set; }
    public ICollection<CustomRoll> CustomRolls { get; set; }

    public Character()
    {
        Items = new List<Item>();
        Skills = new List<Skill>();
        Spells = new List<Spell>();
        CustomRolls = new List<CustomRoll>();
    }

    public Character(string name)
    {
        Name = name;
        Items = new List<Item>();
        Skills = new List<Skill>();
        Spells = new List<Spell>();
        CustomRolls = new List<CustomRoll>();
    }
}

File: client/src/models/Character.ts
interface ICharacter {
    id: number;
    name: string;
    strength: number;
    dexterity: number;
    inteligence: number;
    health: number;
    perception: number;
    will: number;
    magery: number;
    damageReduction: number;
    hitPoints: number;
    fatiguePoints: number;
    energyReserve: number;
}

export default class Character implements ICharacter {
    
    public id: number;
    public name: string;
    public strength: number;
    public dexterity: number;
    public inteligence: number;
    public health: number;
    public perception: number;
    public will: number;
    public magery: number;
    public damageReduction: number;
    public hitPoints: number;
    public fatiguePoints: number;
    public energyReserve: number;

    constructor() {
        this.id = 0;
        this.name = '';
        this.strength = 10;
        this.dexterity = 10;
        this.inteligence = 10;
        this.health = 10;
        this.perception = 10;
        this.will = 10;
        this.magery = 0;
        this.damageReduction = 0;
        this.energyReserve = 0;
        this.hitPoints = 10;
        this.fatiguePoints = 10;
    }
}